runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/AnagramClue.java

```diff
@@ -71,85 +71,85 @@
+		new AnagramClue("GOT A BOY", "Gabooty", new WorldPoint(2790, 3066, 0), "Tai Bwo Wannai", "How many buildings are in the village?", "11"),
 		new AnagramClue("GOBLETS ODD TOES", "Otto Godblessed", new WorldPoint(2501, 3487, 0), "Otto's Grotto", "How many types of dragon are there beneath the whirlpool's cavern?", "2"),
 		new AnagramClue("HALT US", "Luthas", new WorldPoint(2938, 3152, 0), "Banana plantation, Karamja"),
-		new AnagramClue("HEORIC", "Eohric", new WorldPoint(2900, 3565, 0), "Top floor of Burthorpe Castle", "King Arthur and Merlin sit down at the Round Table with 8 knights. How many degrees does each get?", "36"),
-		new AnagramClue("HIS PHOR", "Horphis", new WorldPoint(1639, 3812, 0), "Arceuus Library, Zeah", "On a scale of 1-10, how helpful is Logosia?", "1"),
-		new AnagramClue("I AM SIR", "Marisi", new WorldPoint(1737, 3557, 0), "Allotment patch, South of Hosidius chapel", "How many cities form the Kingdom of Great Kourend?", "5"),
+		new AnagramClue("HEORIC", "Eohric", new WorldPoint(2900, 3565, 0), "Top floor, Burthorpe Castle", "King Arthur and Merlin sit down at the Round Table with 8 knights. How many degrees does each get?", "36"),
+		new AnagramClue("HIS PHOR", "Horphis", new WorldPoint(1639, 3812, 0), "Arceuus Library", "On a scale of 1-10, how helpful is Logosia?", "1"),
+		new AnagramClue("I AM SIR", "Marisi", new WorldPoint(1737, 3557, 0), "Hosidius allotment patch", "How many cities form the Kingdom of Great Kourend?", "5"),
 		new AnagramClue("ICY FE", "Fycie", new WorldPoint(2630, 2997, 0), "East Feldip Hills"),
 		new AnagramClue("I DOOM ICON INN", "Dominic Onion", new WorldPoint(2609, 3116, 0), "Nightmare Zone", "How many reward points does a herb box cost?", "9,500"),
-		new AnagramClue("I EVEN", "Nieve", new WorldPoint(2432, 3422, 0), "The slayer master in Gnome Stronghold", "How many farming patches are there in Gnome stronghold?", "2"),
-		new AnagramClue("IM N ZEZIM", "Immenizz", new WorldPoint(2592, 4324, 0), "The Imp inside Puro-Puro"),
-		new AnagramClue("KAY SIR", "Sir Kay", new WorldPoint(2760, 3496, 0), "The courtyard in Camelot Castle", "How many fountains are there within the grounds of Camelot castle?", "6"),
-		new AnagramClue("LEAKEY", "Kaylee", new WorldPoint(2957, 3370, 0), "Rising Sun Inn in Falador", "How many chairs are there in the Rising Sun?", "18"),
-		new AnagramClue("LARK IN DOG", "King Roald", new WorldPoint(3220, 3476, 0), "Ground floor of Varrock castle", "How many bookcases are there in the palace library?", "24"),
-		new AnagramClue("LOW LAG", "Gallow", new WorldPoint(1805, 3566, 0), "Vinery in the Great Kourend", "How many vine patches can you find in this vinery?", "12"),
+		new AnagramClue("I EVEN", "Nieve", new WorldPoint(2432, 3422, 0), "Gnome Stronghold, near Stronghold Slayer Cave entrance", "How many farming patches are there in Gnome stronghold?", "2"),
```

Comment:
```suggestion
		new AnagramClue("I EVEN", "Nieve", new WorldPoint(2432, 3422, 0), "Gnome Stronghold, near Stronghold Slayer Cave", "How many farming patches are there in Gnome stronghold?", "2"),
```

Reply:
I think the entrance adds relevant context here -- otherwise what does being "near" the cave mean?

Reply:
Maybe "outside" the cave then